example/test/get_dom_node_test.dart

```diff
@@ -41,7 +42,7 @@ class SimpleComponent extends react.Component {
     react.div({}, [
       react.span({"ref": "refToSpan"}, "Test"),
       react.span({}, counter),
-      react.button({"onClick": (_) => (getDOMNode() as HtmlElement).children.first.text = (++counter).toString()},"Increase counter"),
+      react.button({"onClick": (_) => (react_dom.findDOMNode(this) as HtmlElement).children.first.text = (++counter).toString()},"Increase counter"),
```

Comment:
It was deprecated in 0.14.0 ([react-dart 0.9.0](86)) and remove in 15.0.0.

https://facebook.github.io/react/blog/2016/04/07/react-v15.html#removed-deprecations